tests/base.py

```diff
@@ -0,0 +1,28 @@
+
+from mongoengine import connect, disconnect
+from config import config
+from app import create_app
+import os
+
+
+class BaseTest(object):
+
+    def create_app(self):
+        app.config.from_object(config["testing"])
```

Comment:
I just want to understand here, why do we need to create the app here?

Reply:
So that the testing will happen on a different instance of the application